api-reference/beta/api/directory_get.md

```diff
@@ -0,0 +1,74 @@
+# Get deletedItem
+
+Retrieve the properties of an item in [deleted items](../resources/directory.md).
+
+### Prerequisites
+One of the following **scopes** are required to execute this API:
```

Comment:
Again - user support for deletedItems and restore is not there yet - unless Maheedhar added it???
Please remove.